src/backend/base/langflow/components/processing/merge_data.py

```diff
@@ -1,94 +1,92 @@
+from enum import Enum
+
+import pandas as pd
```

Comment:
You shouldn't use `pandas` directly as it won't allow us to interact with Data objects